### PR TITLE
[MERGE] dev-apa into master

### PR DESCRIPTION
**Pull request by AntonSkrub**

### CODE REVIEW

File: pkg/goGit/goGit.go [hunk 1]
```diff
@@ -17,7 +14,7 @@ import (
 func UpdateLocalCopies(names []string, config *config.Config) {
 	for i := 0; i < len(names); i++ {
 		fmt.Printf("Value of i currently is: %v\n", i)
-		if i >= 3 {
+		if i >= 5 {
 			fmt.Println("we break here")
 			break
 		}
```
Comment (valentin-kaiser) [inline pkg/goGit/goGit.go]: Remove the debug print. and why the if condition? If only for development purposes, I would remove these

File: pkg/config/config.go [hunk 2]
```diff
@@ -19,25 +19,26 @@ type Config struct {
 
 	OutputPath string `yaml:"OutputPath"`
 
-	EnableLog bool `yaml:"EnableLog"`
-	LogLevel  int  `yaml:"LogLevel"`
+	ListRefereces    bool `yaml:"ListRefereces"`
```
Comment (valentin-kaiser) [inline pkg/config/config.go]: Misspelling: Refereces -> References

File: pkg/gitapi/gitapi.go [hunk 3]
```diff
@@ -41,16 +40,16 @@ func GetList(config *config.Config) []string {
...
 	var repoNames []string
-	logr.Println("[GitAPI] Found Repositories:")
 	for _, repo := range repos {
 		i++
 		name, ok := repo["name"].(string)
 		if !ok {
 			logr.Errorf("[GitAPI] failed converting the repository name to string: %v\n", err)
 		} else {
 			repoNames = append(repoNames, name)
-			fmt.Printf("%v. %v%v\n", i, config.OrgaName, repo["name"])
```
Comment (valentin-kaiser) [inline pkg/gitapi/gitapi.go:52]: The code is not relevant in any way. If you remove debug prints, please do it properly

File: pkg/goGit/log.go [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+	fmt.Println("------ tag references --------")
+	if len(tagList) == 0 {
+		fmt.Println("--- No tags found ---")
+	} else {
+		for _, tag := range tagList {
+			tagName := tag[len(tagRefPrefix):]
+			fmt.Println(tagName)
+		}
+	}
```
Comment (valentin-kaiser) [inline pkg/goGit/log.go]: Mixed logrus and fmt.Print usage. Unify to one of both

**Review by valentin-kaiser — CHANGES_REQUESTED** [review]

File: pkg/goGit/goGit.go (hunk 1 above)
Comment (AntonSkrub) [reply pkg/goGit/goGit.go]: Resolved in [76b3090](https://github.com/AntonSkrub/GoGit-Integration/pull/6/commits/76b3090a4275bf0725703d0684da0ca154dd884f)

File: pkg/config/config.go (hunk 2 above)
Comment (AntonSkrub) [reply pkg/config/config.go]: Resolved in [76b3090](https://github.com/AntonSkrub/GoGit-Integration/pull/6/commits/76b3090a4275bf0725703d0684da0ca154dd884f)

File: pkg/gitapi/gitapi.go (hunk 3 above)
Comment (AntonSkrub) [reply pkg/gitapi/gitapi.go:52]: Resolved in [76b3090](https://github.com/AntonSkrub/GoGit-Integration/pull/6/commits/76b3090a4275bf0725703d0684da0ca154dd884f)

File: pkg/goGit/log.go (hunk 4 above)
Comment (AntonSkrub) [reply pkg/goGit/log.go]: Resolved in [76b3090](https://github.com/AntonSkrub/GoGit-Integration/pull/6/commits/76b3090a4275bf0725703d0684da0ca154dd884f)

**Review by valentin-kaiser — APPROVED** [review]

**Review by Plaenkler — APPROVED** [review]

**Comment by valentin-kaiser** [comment]
Will this be merged?

**Review by valentin-kaiser — APPROVED** [review]

**Pull request merged**